README.md

```
@@ -1,4 +1,4 @@
-<div align="center"><img src="./assets/icon.png" width="120" alt="CodeSandbox Theme Logo" />
+<div align="center"><img src="./assets/icon.png" width="120" alt="Dark Emerald Theme Logo" />
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: div

(MD033, no-inline-html)